README.md

```diff
@@ -2,34 +2,38 @@
 
 [![Build Status](https://travis-ci.org/onnx/onnx-coreml.svg?branch=master)](https://travis-ci.org/onnx/onnx-coreml)
 
-This tool converts [ONNX](https://onnx.ai/) models to Apple Core ML format. To convert Core ML models to ONNX, use [ONNXMLTools](https://github.com/onnx/onnxmltools).
+## PyTorch Models
 
-There's a comprehensive [Tutorial](https://github.com/onnx/tutorials/tree/master/examples/CoreML/ONNXLive/README.md) showing 
-how to convert PyTorch style transfer models through ONNX to Core ML models and run them in an iOS app. 
+For converting PyTorch models to CoreML format, the recommended approach is to use **new** PyTorch to Core ML converter, introduced in the [`coremltools 4.0`](https://github.com/apple/coremltools) python package.
 
-To get the latest version of `onnx-coreml` from PyPI:
+Please see the  coremltools documentation on [PyTorch conversion](https://coremltools.readme.io/docs/pytorch-conversion) for example usage.
+
+To get the latest version of coremltools:
 
 ```shell
-pip install --upgrade onnx-coreml
-pip install --upgrade coremltools # onnx-coreml depends on the coremltools package
+pip install coremltools==4.0b1
 ```
 
-For the latest changes please see the [release notes](https://github.com/onnx/onnx-coreml/releases).
+For the latest changes in coremltools, please see the [release notes](https://github.com/apple/coremltools/releases/).
+
+## ONNX Models
 
-To get the latest version from source (master branch of this repository), please see the [installation section](#Installation).
+Code for ONNX to Core ML conversion is now available through `coremltools` python package. For access to bug fixes, community support and requests, please use [coremltools](https://github.com/apple/coremltools) github repository. 
 
-## Usage
+For ONNX conversion `coremltools.converters.onnx.convert` is the only supported API.
 
-Please see the ONNX conversion section in the [Neural network guide](https://github.com/apple/coremltools/blob/master/examples/NeuralNetworkGuide.md) 
-on how to use the converter. 
+`onnx-coreml` package will **no longer be maintained**.
 
-There are a few [notebook examples](https://github.com/apple/coremltools/tree/master/examples/neural_network_inference) 
-as well for reference.  
+Note: ONNX converter is not under any active feature development.
+
+## ONNX Conversion API
 
 
 ### Parameters
 
 ```python
+from coremltools.converters.onnx import convert
```

Comment:
All of this code should live in the documentation of coremltools.readme.io

Reply:
Moved.